src/components/PledgeCollectiveCard.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { ExternalLinkAlt } from 'styled-icons/fa-solid/ExternalLinkAlt';
+import { get } from 'lodash';
+
+// import { defaultImage, defaultBackgroundImage } from '../constants/collectives';
+
+// import { imagePreview } from '../lib/utils';
+
+import { Flex, Box } from '@rebass/grid';
+import Container from './Container';
+import { P } from './Text';
+import { Link } from '../server/pages';
+import StyledLink from './StyledLink';
+import { themeGet } from 'styled-system';
+import Currency from './Currency';
+
+const defaultPledgedLogo = '/static/images/default-collective-logo.svg';
+
+const CollectiveLogoContainer = styled(Flex)`
+  justify-content: center;
+  border-top: 1px solid ${themeGet('colors.black.300')};
+`;
+
+class PledgeCollectiveCard extends React.Component {
+  static propTypes = {
+    membership: PropTypes.object,
+    LoggedInUser: PropTypes.object,
+    collective: PropTypes.shape({
+      backgroundImage: PropTypes.string,
+      description: PropTypes.string,
+      image: PropTypes.string,
+      name: PropTypes.string.isRequired,
+      slug: PropTypes.string.isRequired,
+      currency: PropTypes.string.isRequired,
+    }).isRequired,
+    backers: PropTypes.shape({
+      totalAmount: PropTypes.number,
+    }),
+  };
+
+  render() {
+    const { collective } = this.props;
+
+    let website = collective.website;
+    if (!website && collective.githubHandle) {
+      website = `https://github.com/${collective.githubHandle}`;
+    }
+
+    return (
+      <Container
+        bg="white"
+        borderRadius="8px"
+        border="1px solid rgba(18,19,20,0.2)"
+        display="flex"
+        flexDirection="column"
+        justifyContent="space-between"
+        minHeight="100%"
+        overflow="hidden"
+      >
+        <CollectiveLogoContainer mt={52}>
+          <Box mt={-50}>
+            <Link route="collective" params={{ slug: collective.slug }}>
+              <img src={defaultPledgedLogo} alt="Pledged Collective" radius={8} width="94px" mb={-43} />
+            </Link>
+          </Box>
+        </CollectiveLogoContainer>
+
+        <P fontSize="1.4rem" textAlign="center" fontWeight="bold" mb={-1} color="black.600">
+          <Link route="collective" params={{ slug: collective.slug }} color="black.600">
+            {collective.name}
+          </Link>
+        </P>
+
+        <P fontSize="1.2rem" textAlign="center" p={1}>
+          <FormattedMessage id="Pledgecollective.card" defaultMessage="PLEDGED COLLECTIVES" />
+        </P>
+        <Link route="createCollectivePledge" params={{ slug: collective.slug }} passHref>
+          <StyledLink href={website} color="primary.500" fontSize="Caption">
```

Comment:
Why does this link has a `href`? The route is already provided by `Link route="createCollectivePledge"`